10.9-libcxx/stable/main/finkinfo/x11/xft2-dev.info

```diff
@@ -67,7 +63,7 @@ SplitOff: <<
 	<<
 	Files: lib/xft2/lib/libXft.2.*dylib
 	Shlibs: %p/lib/xft2/lib/libXft.2.dylib 5.0.0 %n (>= 2.2.0-1)
```

Comment:
```suggestion
	Shlibs: %p/lib/xft2/lib/libXft.2.dylib 6.0.0 %n (>= 2.3.8-1)
```
need this to pass validation

Reply:
Done.  Should probably spend more time studying SHlibs syntax